### PR TITLE
Add RUNBOOK.md

### PR DESCRIPTION
**Pull request by chee**

### CODE REVIEW

File: RUNBOOK.md [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+origami-team
+
+## Known About By
+
+* jake.champion
+* rowan.manning
+
+## Dependencies
+
+* ft-fastly
```
Comment (JakeChampion) [inline RUNBOOK.md]: I think we should add Cloudinary and Content API here

File: RUNBOOK.md [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+4. Cloudinary responds with a cached image if it exists [end], or fetches the image from its origin
+5. The origin service responds to Cloudinary with an image or 404
+
+
+## More Information
+
+<table width="100%">
+<tbody>
+<tr>
+<td style="padding: 10px; border: 1px solid; vertical-align: top; text-align: left;"><a href="https://github.com/Financial-Times/origami-image-service#readme" target="_blank" rel="nofollow noopener">https://github.com/Financial-Times/origami-image-service#readme</a></td>
```
Comment (JakeChampion) [inline RUNBOOK.md]: Should this be a link instead of a table containing a link?

File: RUNBOOK.md [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+
+Our Fastly config automatically routes requests between the production EU and US Heroku applications. If one of those regions is down, Fastly will route all requests to the other region.
+
+## Data Recovery Details
+
+No data-recovery required.
+
+## Release Details
+
+The application is deployed to QA whenever a new commit is pushed to the `master` branch of this repo on GitHub. To release to production, the QA application must be [manually promoted through the Heroku interface](https://dashboard.heroku.com/pipelines/748923ac-b3c0-4289-a0ac-c26b5a7dbe3a).
```
Comment (JakeChampion) [inline RUNBOOK.md]: The release process has changed, we deploy to production based on a Semver tagged commit.

https://github.com/Financial-Times/origami-image-service/blob/master/.circleci/config.yml#L175

File: RUNBOOK.md [hunk 4]
```diff
@@ -44,17 +36,17 @@ origami-team
...
 
-* jake.champion
-* rowan.manning
+- jake.champion
+- rowan.manning
 
 ## Dependencies
 
-* ft-fastly
+- ft-fastly
```
Comment (JakeChampion) [inline RUNBOOK.md]: We need to add dyn, Heroku, and Cloudinary

File: RUNBOOK.md (hunk 1 above)
Comment (chee) [reply RUNBOOK.md]: i don't think Cloudinary is in biz-ops yet, should we add it??

**Pull request merged**